src/stanc/stanc.ml

```diff
@@ -190,6 +191,9 @@ let options =
     ; ( "-fsoa"
       , Arg.Unit (fun () -> soa_opt := true)
       , "\tTurn on the Struct of Arrays optimization" )
+    ; ( "-fopencl"
+      , Arg.Unit (fun () -> opencl_opt := true)
+      , "\tTurn on OpenCL optimization" )
```

Comment:
It's unfortunate that it would disagree with the `-fsoa` flag, but I personally prefer the `--use-opencl` name.

Reply:
That's fine I can switch this back.

It makes sense that it is one or the other since `--use-opencl` should use only the GPU in log_prob. So no chance to use SoA